lib/cartopy/examples/gridliner.py

```diff
@@ -0,0 +1,40 @@
+"""
+Gridlines and tick Labels
+-------------------------
+
+Theses examples demonstrate how to quickly add longitude
+and latitude gridlines and tick labels on a non-rectangular projection.
+
+As you can see on the first example,
+longitude labels may be drawn on left and right sides,
+and latitude labels may be drawn on bottom and top sides.
+Thanks to the ``dms`` keyword, minutes are used when appropriate
+to exprimate fractions of degree.
```

Comment:
exprimate? I don't know what word this is supposed to be.

Reply:
exprimante -> display?